utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -29,11 +30,12 @@ class HtmlParser(private val context: Context, private val entityType: String, p
       );
```

Comment:
Remove `;`

Reply:
Remove `;`

in 

`htmlContent = htmlContent.replace(CUSTOM_IMG_TAG, REPLACE_IMG_TAG, /* ignoreCase= */false);`

Reply:
done.